perfkitbenchmarker/providers/ibmcloud/flags.py

```diff
@@ -0,0 +1,121 @@
+
+flags.DEFINE_string('ibmcloud_image_username', 'root',
+                    'Ssh username for cloud image.')
+
+flags.DEFINE_integer('ibmcloud_polling_delay', 2,
+                     'Delay between polling attempts for action objects.')
+
+flags.DEFINE_string('ibmcloud_login_validfrom', None,
+                    'Valid from for token creation.')
+
+flags.DEFINE_integer('ibmcloud_sleep_post_login', 5,
+                     'Sleep for some seconds after login.')
+
+flags.DEFINE_string('ibmcloud_speccpu_tarfile', None,
+                    'tarball for speccpu2017.')
+
+flags.DEFINE_integer('ibmcloud_timeout', 600,
+                     'timeout in secs.')
+
+flags.DEFINE_integer('ibmcloud_boot_disk_size', 10,
+                     'boot volume disk size.')
+
+flags.DEFINE_boolean('ibmcloud_numa_pin', False,
+                     'numaPin flag.')
+
+flags.DEFINE_string('ibmcloud_cpu_family', None,
+                    'one of allowed cpu family.')
+
+flags.DEFINE_boolean('ibmcloud_debug', False,
+                     'debug flag.')
+
+flags.DEFINE_boolean('ibmcloud_resources_keep', False,
+                     'keep resources.')
+
+flags.DEFINE_string('ibmcloud_profile', 'cx2-2x4',
+                    'same as machine_type')
+
+flags.DEFINE_string('ibmcloud_volume_profile', 'custom',
+                    'volume profile')
+
+flags.DEFINE_string('ibmcloud_bootvol_encryption_key', None,
+                    'boot volume encryption key crn')
+
+flags.DEFINE_string('ibmcloud_datavol_encryption_key', None,
+                    'data volume encryption key crn')
+
+flags.DEFINE_string('ibmcloud_vpcid', None,
+                    'IBM Cloud vpc id')
+
+flags.DEFINE_string('ibmcloud_subnet', None,
+                    'primary subnet id')
+
+flags.DEFINE_string('ibmcloud_networks', None,
+                    'additional network ids, comma separated')
+
+flags.DEFINE_string('ibmcloud_prefix', 'perfkit',
+                    'resource name prefix')
+
+flags.DEFINE_string('ibmcloud_rgid', None,
+                    'Resource Group id for the account.')
+
+flags.DEFINE_integer('ibmcloud_boot_volume_iops', 3000,
+                     'boot voume iops')
+
+flags.DEFINE_integer('ibmcloud_boot_volume_size', 0,
+                     'boot voume size in GB')
+
+flags.DEFINE_string('ibmcloud_pub_keyid', None,
+                    'rias public sshkey id')
+
+flags.DEFINE_string('ibmcloud_priv_keyfile', None,
+                    'full path to ssh key file containing private key.')
+
+flags.DEFINE_integer('ibmcloud_network_mtu', 9000,
+                     'MTU size on network interfaces.')
+
+flags.DEFINE_string('ibmcloud_image_name', None,
+                    'image name')
+
+flags.DEFINE_integer('ibmcloud_subnets_extra', 0,
+                     'extra subnets to lookup')
+
+flags.DEFINE_integer('ibmcloud_vdisks_extra', 0,
+                     'extra disks to create')
+
+flags.DEFINE_string('ibmcloud_preprovisioned_data', None,
```

Comment:
The other ones are called "(cloudname)_preprovisioned_data_bucket"  can you change to that format?

Reply:
changed the name.